core/src/main/java/tech/tablesaw/api/Table.java

```diff
@@ -1069,15 +1069,48 @@ public boolean hasNext() {
     };
   }
 
+  /**
+   * Iterates over rolling consecutive rows
+   *
```

Comment:
I'm not entirely clear, but it looks like this is stepping forward (e.g. if n = 2, the first call to next() gets rows 0 and 1, and the second gets rows 2 and 3). Is that it, or is it rolling, such that the first call to next() gets rows 0 and 1, and the second gets rows 1 and 2? 

Either way could we add that kind of example to the comment so it's clear without reading the code?

Reply:
I've renamed the methods and added comments